### PR TITLE
Switch to Gradle build system (extends #230)

### PR DESCRIPTION
**Pull request by skywave**
This fixes the lint errors that were blocking #230

### CODE REVIEW

File: library/src/main/res/values-v9/strings.xml [hunk 1]
```diff
@@ -2,5 +2,5 @@
 <!-- Copyright 2012 Square, Inc. -->
 
 <resources>
-  <string name="month_name_format">LLLL yyyy</string>
+  <string name="month_name_format" translatable="false">LLLL yyyy</string>
```
Comment (edenman) [inline library/src/main/res/values-v9/strings.xml]: revert this: date format strings should be translatable so the keys can be reordered in different languages

File: library/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <!-- Copyright 2012 Square, Inc. -->
 
 <resources>
-  <string name="day_name_format">EEE</string>
+  <string name="day_name_format" translatable="false">EEE</string>
```
Comment (edenman) [inline library/src/main/res/values/strings.xml]: same thing in this file

**Comment by edenman** [comment]
LG, thanks for doing this!  Please:

- Revert the `translatable` changes
- Squash into a single commit
- Fill out Square's [CLA](https://docs.google.com/a/squareup.com/forms/d/13WR8m5uZ2nAkJH41k7GdVBXAAbzDk00vxtEYjd6Imzg/viewform?formkey=dDViT2xzUHAwRkI3X3k5Z0lQM091OGc6MQ)

**Comment by skywave** [comment]
CLA signed and untranslatable reverted. The then failing build was fixed by disabling the lint checks on these two resources.

File: library/src/main/res/values-v9/strings.xml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!-- Copyright 2012 Square, Inc. -->
 
-<resources>
-  <string name="month_name_format">LLLL yyyy</string>
+<resources xmlns:tools="http://schemas.android.com/tools">
+  <string name="month_name_format" tools:ignore="MissingTranslation">LLLL yyyy</string>
```
Comment (JakeWharton) [inline library/src/main/res/values-v9/strings.xml]: I think I would prefer if you reverted this and downgraded the check to a warning in the `lintOptions` of the `build.gradle` instead.

Comment (skywave) [inline library/src/main/res/values-v9/strings.xml]: Sure; that would work too; done.

**Comment by edenman** [comment]
Cool: only thing left is to squash everything into a single commit.

File: sample/src/main/java/com/squareup/timessquare/sample/SampleTimesSquareActivity.java [hunk 4]
```diff
@@ -25,7 +25,7 @@
 import static android.widget.Toast.LENGTH_SHORT;
 
 public class SampleTimesSquareActivity extends Activity {
-  private static final String TAG = "SampleTimesSquareActivity";
+  private static final String TAG = "SampleTimesSquareActivi";
```
Comment (edenman) [inline sample/src/main/java/com/squareup/timessquare/sample/SampleTimesSquareActivity.java]: was this intentional?  missed it on the first pass.

Comment (edenman) [inline sample/src/main/java/com/squareup/timessquare/sample/SampleTimesSquareActivity.java]: nm, 23char limit on log tags, carry on.

**Pull request closed without merging**